src/views/Rankings.vue

```diff
@@ -382,6 +390,16 @@ export default class RankingsView extends Vue {
   public async setLeague(league: number) {
     await this.$store.direct.dispatch.rankings.setLeague(league);
   }
+
+  public playerIsRanked(rank: Ranking): boolean {
+    return rank.player.games > 0        
+  }
+
+  public routeToProfilePage(playerId: string) {
```

Comment:
can you extract this in this file https://github.com/w3champions/w3champions-ui/blob/master/src/helpers/url-functions.ts